docs/docs/getting-started/installation.mdx

```diff
@@ -47,16 +47,16 @@ Checkout the latest stable version [here](https://mapperly.riok.app).
 
 </Conditional>
 
-:::info
+### Removing runtime dependency
+
 It is possible to remove Mapperly runtime references completely by setting `PrivateAssets="all"` and `ExcludeAssets="runtime"` on the `PackageReference`.  
 `PrivateAssets="all"` ensures that projects referencing this project do not also get a reference to `Riok.Mapperly`.  
 `ExcludeAssets="runtime"` ensures that the Mapperly .dll files are not copied to the build output.
 
 Removing the runtime dependency comes with the disadvantage that whenever the runtime tries
 to load a referenced attribute from `Riok.Mapperly.Abstractions` a `FileNotFoundException` is thrown.
-This is, for example, the case if `typeof(MyMapper).GetCustomAttributes()` is called.
+This is, for example, is the case if `typeof(MyMapper).GetCustomAttributes()` is called.
```

Comment:
Sounds weird. Something like `This is for example the case when ... is called.` is IMO better.

Reply:
What did I do here, reverted